core/src/geometry/mod.rs

```diff
@@ -94,6 +94,7 @@ where
 
 /// Anchor point.
 #[derive(Debug, Ord, PartialOrd, Eq, PartialEq, Hash, Copy, Clone)]
+#[cfg_attr(feature = "defmt_support", derive(::defmt::Format))]
```

Comment:
There are three different variants of this attribute in this PR:
```rust
#[cfg_attr(feature = "defmt", derive(::defmt::Format))]
#[cfg_attr(feature = "defmt", derive(defmt::Format))]
#[cfg_attr(feature = "defmt_support", derive(::defmt::Format))]
```
Could you use the same variant everywhere?

Reply:
Note to self: use `defmt_support` as the name, and `derive(::defmt::Format)` as the path, since it is a little bit more specific than `derive(defmt::Format)`.